command/CommandController.js

```diff
@@ -737,12 +743,29 @@ async function searchForPocketArticleByUrl(getBody, url, extendedData) {
       if (keysArr.length > 0) {
         let isArticleId = 0;
```

Comment:
This variable name is confusing since it's named like a boolean. Something like `listIndex` would read easier.

Reply:
Agreed.  Changing.